src/main/java/ua/danit/final_project/entities/Location.java

```diff
@@ -0,0 +1,40 @@
+package ua.danit.final_project.entities;
+
+import lombok.Data;
+
+import javax.persistence.CascadeType;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.util.List;
+
+@Entity
+@Table(name = "location")
+@Data
+public class Location implements Serializable {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @Column(name = "l_title", nullable = false)
+  private String title;
+
+  @Column(name = "l_info")
+  private String info;
+
+  @ManyToMany(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
+  @JoinTable(
+          name = "task_location",
+          joinColumns = {@JoinColumn(name = "l_id")},
+          inverseJoinColumns = {@JoinColumn(name = "t_id")})
+  List<Task> tasks;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "tasks" private or transient. ([squid:S1948](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1948))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=82&issues=AWVYzZew2xJdnw6Qj_eR&open=AWVYzZew2xJdnw6Qj_eR)

[ISSUE_KEY]: AWVYzZew2xJdnw6Qj_eR